### ISSUE TITLE
PlantUML diagrams lines misplaced

### ISSUE THREAD

**Issue by kohtala** [Kozea/CairoSVG#266]
I'm using [PlantUML](https://plantuml.com/) for diagramming.

Unfortunately some lines are misplaced in some diagrams.

I have CairoSVG 2.4.2 and did try `pip install --upgrade https://github.com/Kozea/CairoSVG/archive/master.zip` to see if commit 1b3d44858b would have a fix. They worked the same.

Host is Ubuntu 18.04.4 LTS. Python 3.7.5. plantuml 1:1.2017.15-1, but I also tried agains svg produced by 1.2020.5 from the jar.

I get this:
![deployment](https://user-images.githubusercontent.com/1055533/77626795-8eac2500-6f4e-11ea-85bd-633a6fe0923f.png)

I really need PDF, which looks much the same [deployment.pdf](https://github.com/Kozea/CairoSVG/files/4386046/deployment.pdf)

Other tools (this with rsvg-convert) produce this:
![deployment-rsvg](https://user-images.githubusercontent.com/1055533/77627117-12fea800-6f4f-11ea-8fb5-fcc656885106.png)

The images were created with these plantuml and svg files:
[deployment.zip](https://github.com/Kozea/CairoSVG/files/4386043/deployment.zip)

``` shell
$ plantuml -tsvg deployment.plantuml
$ cairosvg -fpdf -o deployment.pdf deployment.svg
$ cairosvg -fpng -o deployment.png deployment.svg
```

Thanks. It looks promising. I did missinglinkelectronics/sphinxcontrib-svg2pdfconverter#7 to hopefully give it more visibility and use.

**Comment by liZe** [comment]
Hello, thanks for the bug report!

The bug is caused by the gaussian blur filter. This filter is supposed to move the background rectangle (+4, +4), to blur it and to blend the blur with the original rectangle. As CairoSVG doesn’t support complex filters (see #23), it just moves the rectangle.

A simple workaround is to remove the filter definition: you can remove the `<defs>` or the `<filter>` tag from the SVG file. Handling complex filters is almost impossible using Cairo (see Kozea/WeasyPrint#13), so you can’t expect a quick fix for CairoSVG :unamused:.

**Comment by kohtala** [comment]
Thanks.

With your hint to shadows, I found PlantUML has directive

    skinparam shadowing false

This removes the shadow and the result from cairosvg looks much better.

![deployment](https://user-images.githubusercontent.com/1055533/77730226-0b033e80-7009-11ea-81c8-0625cfc2fb05.png)

**Comment by kohtala** [comment]
I closed this as duplicate of #23. No point in keeping this open. Thanks.
